### ISSUE TITLE
(Optionally) Disable sector lighting effects

### ISSUE THREAD

**Issue by caligari87** [caligari87/darkdoomz#2]
From Phantombeta on the ZDoom Discord, also suggested by marisa:
```

        ThinkerIterator it = ThinkerIterator.Create ("Lighting");
        Lighting effect;

        while (effect = Lighting (it.Next ()))
            effect.Destroy ();
```

**Comment by caligari87** [comment]
Closed by a2246c7.

**Comment by caligari87** [comment]
Revisit this, should make it optional (with a warning that a restart may be required to re-enable disabled effects)
